serverengine.gemspec

```diff
@@ -20,6 +20,9 @@ Gem::Specification.new do |gem|
   gem.required_ruby_version = ">= 1.9.3"
 
   gem.add_dependency "sigdump", ["~> 0.2.2"]
+  if /mswin|mingw/ =~ RUBY_PLATFORM
```

Comment:
Does this `if` work when user install serverengine.gem on a windows machine?
To make this work, Rakefile need to create 2 gems (serverengine-VERSION.gem and serverengine-VERSION-mingw.gem), right? I think that is unnecessary optimization because installing win32-pipe does nothing on non-windows platform. So, why don't you simply depend on win32-pipe without `if`?